### PR TITLE
document how to use extension on Windows

### PR DESCRIPTION
**Pull request by ulugbekna**
Workflow for esy is missing though

### CODE REVIEW

**Comment by mnxn** [comment]
Looks good.

**Review by mnxn — APPROVED** [review]

**Comment by mnxn** [comment]
I added a few tweaks on master to make it easier to read. Esy workflow can be another PR.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,24 @@ _Please report any bugs you encounter._
...
+If you are on Windows, there are two ways to use this extension:
+
+1. Launch VSCode from cygwin. This will let you choose a global or opam sandbox.
+
+2. Use a custom sandbox with
+   https://fdopen.github.io/opam-repository-mingw/ocaml-env/. If it is on your
+   path you can input `ocaml-env exec -- $prog $args` for the custom command
+   template, or manually edit the `ocaml.sandbox` setting so it is
+
+```javascript
```
Comment (smorimoto) [inline README.md:28]: Isn't this json?

Comment (ulugbekna) [reply README.md:28]: My omission, but there is no difference in rendering/color-highlighting anyway, as far as I can tell.

**Comment by ulugbekna** [comment]
> I added a few tweaks on master to make it easier to read. Esy workflow can be another PR.

@mnxn thanks! I don't know anything about esy though so can't help here.

**Pull request merged**